### PR TITLE
fixed -p to -i in find-dependencies

### PR DESCRIPTION
**Pull request by jochman**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/29684

## Must have
- [x] Tests

### CODE REVIEW

File: demisto_sdk/__main__.py [hunk 1]
```diff
@@ -939,21 +940,26 @@ def update_pack_releasenotes(**kwargs):
     '-h', '--help'
 )
 @click.option(
-    "-p", "--pack_folder_name", help="Pack folder name to find dependencies.", required=True)
+    "-i", "--input", help="Pack folder name to find dependencies. For example: Pack/HelloWorld", required=True,
```
Comment (yaakovi) [inline demisto_sdk/__main__.py:943]:
```suggestion
    "-i", "--input", help="Pack path to find dependencies. For example: Pack/HelloWorld", required=True,
```

File: demisto_sdk/__main__.py [hunk 2]
```diff
@@ -939,21 +940,26 @@ def update_pack_releasenotes(**kwargs):
...
-
+def find_dependencies_command(id_set_path, verbose, no_update, **kwargs):
+    update_pack_metadata = not no_update
+    input_path: Path = kwargs["input"]  # To not shadow python builtin `input`
+    try:
+        assert "Packs/" in input_path
+        pack_name = str(input_path).replace("Packs/", "")
+        assert "/" not in pack_name
+    except AssertionError:
+        print_error("Input path is not a pack. For example: Pack/HelloWorld")
```
Comment (yaakovi) [inline demisto_sdk/__main__.py:961]: add a test for invalid input

**Review by yaakovi — COMMENTED** [review]
looks good

**Comment by roysagi** [comment]
@jochman - please change in the command README file as well.

**Review by yaakovi — APPROVED** [review]

**Pull request merged**